tests/x11/gdm_session_switch.pm

```diff
@@ -58,14 +77,18 @@ sub run {
     send_key "alt-l";
     wait_still_screen;
     send_key "alt-o";
-    assert_screen "displaymanager";
+    assert_screen "displaymanager", 60;
     send_key "ret";
     assert_screen "originUser-login-dm";
     type_password;
     assert_and_click "displaymanager-settings";
     assert_and_click "dm-gnome";
     send_key "ret";
     assert_screen "generic-desktop", 120;
+
+    # Log out and switch to SLE classic
+    $self->prepare_sle_classic;
```

Comment:
this is done twice in this test, also on line 33-34

Reply:
Yes, this is intentional test that GDM session switching still works after logging out of non-classic Gnome.

Reply:
Can you then update also summary, because what is there doesn't describe what is done now.

Reply:
Added a line about rebooting to the summary comment in gdm_session_switch.pm.